build.xml

```diff
@@ -8,26 +8,45 @@
 
 
 <project name="xspec" default="xspec">
-	<description>XSpec is a Behavior Driven Development (BDD) framework for XSLT and XQuery.
-
-Usage:
-
-ant -Dxspec.xml=filename
-
-where:
-
-  filename          XSpec tests XML file
-
-
-</description>
-
+  <description>XSpec is a Behavior Driven Development (BDD) framework for XSLT and XQuery.
+    
+    Usage:
+    
+    ant -Dxspec.xml=filename
+    
+    where:
+    
+    filename          XSpec tests XML file
+    
+    ---
+    
+    OUP patch for running XSpec 0.6.0 as an Oxygen XSLT transformation scenario
+    
+    This file is an updated copy of Oxygen's ${frameworksDir}/xspec/build.xml 
+    
+    Setup:
```

Comment:
These instructions were very useful for the review. When the pull request is merged into `master` and documentation is added to the wiki I suggest to remove these comments and put a link to the wiki page instead.